membership-common/conf/touchpoint.PROD.conf

```diff
@@ -67,7 +67,7 @@ touchpoint.backend.environments {
                     digipack="2c92a0fb4edd70c8014edeaa4ddb21e7"
                     supporterPlus="8a12865b8219d9b4018221061563643f"
                     tierThree="8a1295998ff2ec180190024b287b64c7"
-                    guardianLight="8a128dfb92c341720192dd19626429dc"
+                    guardianAdLite="8a128dfb92c341720192dd19626429dc"
```

Comment:
I guess this is a product ID here? It probably needs updating since Ad-Lite is a new product in Zuora (not a rename of a the existing Light product).

Reply:
@david-pepper the new Product Id is `8a1285e294443da501944b04cb382c9a`

Reply:
Done